en/peripherals/dshot.md

```diff
@@ -2,14 +2,24 @@
 
 DShot is an alternative ESC protocol that has several advantages over PWM or OneShot:
 - Reduced latency
-- Checksum (increased robustness)
-- Digital encoding (no need for ESC calibrations)
-- Possibility to get feedback (telemetry)
-- Sending commands (such as reversing the motor spin direction)
+- Increased robustness via a checksum
+- No need for ESC calibration as mechanism uses digital encoding
+- Telemetry feedback is available/supported on some ESCs
+- Can reverse motor spin directions via commands when needed (rather than physically moving wires/re-soldering).
+- Other useful commands are supported.
+
+This topic shows how to connect and configure DShot ESCs.
+
+
+## Wiring/Connections {#wiring}
 
 DShot ESCs are connected and wired the same way as [PWM ESCs](pwm_escs_and_servo.md), and you can switch between these protocols just by changing software parameters (ESCs automatically detect the selected protocol on startup).
 
-## Configuration
+On Pixhawk flight controllers that have both MAIN and AUX ports, DShot can only be used on the MAIN ports if the AUX port is disabled (see configuration below).
```

Comment:
> DShot can only be used on the MAIN ports if the AUX port is disabled (see configuration below).

Technically this is not correct: the SYS_USE_IO=0 method is only one way of doing it. You could also change the mixer for AUX to add multi-rotor outputs for example, and then still use the MAIN pins for whatever you want.

> DShot can only be used on the MAIN ports

Same as below: it is only used on the AUX ports (if you are refering to the pin labels of the board).

Reply:
Thanks @bkueng , I think I get the distinction now - waiting on your confirmation in https://github.com/PX4/px4_user_guide/pull/578#discussion_r328026199

>> DShot can only be used on the MAIN ports if the AUX port is disabled (see configuration below).

So changing my terminology (avoiding using the term mixer below)...is it reasonable?

A Pixhawk board that has both FMU and IO will label them as AUX and MAIN respectively. Normally you would connect your ESCs to the ports labelled MAIN, but DShot can only be used on the FMU ports (labeled `AUX`). In order to use these boards for controlling motors, you will typically set `SYS_USE_IO=0` (which makes the ports labeled `AUX` behave as though they were the ports labeled `MAIN`) and connect your ESCs to the corresponding AUX-labeled outputs. 

> Technically this is not correct: the SYS_USE_IO=0 method is only one way of doing it. You could also change the mixer for AUX to add multi-rotor outputs for example, and then still use the MAIN pins for whatever you want.

OK, I get that, but probably a bit hard for most.

Reply:
>  So changing my terminology (avoiding using the term mixer below)...is it reasonable?

Yes, looks good.

> OK, I get that, but probably a bit hard for most.

Right, we don't need to explain it, I just want to ensure we know about it.